src/views/jobs/moderator/moderator.jsx

```diff
@@ -0,0 +1,94 @@
+var React = require('react');
+var render = require('../../lib/render.jsx');
```

Comment:
All of these paths are off my one level as you are nested inside of the `jobs` view. Were you able to get this to build locally?